composer.json

```diff
@@ -1,17 +1,29 @@
 {
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/mozilla-l10n/phpclass-bugzilla"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/mozilla-l10n/phpclass-cache"
+        }
+    ],
+
     "require": {
-        "atoum/atoum" : "dev-master",
+        "atoum/atoum": "dev-master",
+        "mozillal10n/bugzilla": "0.*",
+        "mozillal10n/cache": "0.*",
         "sepia/po-parser": "3.0.5"
     },
 
     "require-dev": {
-        "raveren/kint" : "dev-1.0.0-wip"
+        "raveren/kint": "dev-1.0.0-wip"
```

Comment:
we could take this opportunity tu use var_dumper instead of kint like on Transvision